apps/site/.storybook/main.ts

```diff
@@ -1,31 +1,60 @@
-import type { StorybookConfig } from '@storybook/nextjs';
-import classNames from 'classnames';
+import { resolve } from 'node:path';
 
-const rootClasses = classNames(
-  // note: this is hard-coded sadly as next/font can only be loaded within next.js context
-  '__variable_open-sans-normal',
-  // note: this is hard-coded sadly as next/font can only be loaded within next.js context
-  '__variable_ibm-plex-mono-normal'
-);
+import type { StorybookConfig } from '@storybook/react-webpack5';
 
 const config: StorybookConfig = {
   stories: ['../components/**/*.stories.tsx'],
   logLevel: 'error',
   staticDirs: ['../public'],
   typescript: { reactDocgen: false, check: false },
   core: { disableTelemetry: true, disableWhatsNewNotifications: true },
-  framework: {
-    name: '@storybook/nextjs',
-    options: { builder: { useSWC: true } },
-  },
+  framework: '@storybook/react-webpack5',
+  swc: () => ({
+    jsc: {
+      transform: {
+        react: {
+          runtime: 'automatic',
+        },
+      },
+    },
+  }),
   previewBody:
+    '<link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono&family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap" rel="stylesheet">' +
```

Comment:
Maybe worth adding inline line docs explaining what these additions are (including the style ones below) and why they are needed